hs_core/templates/pages/create-resource.html

```diff
@@ -39,8 +39,13 @@
             <h2>Select a resource type and upload files as needed to create a new resource</h2>
 
             <ul>
-            <li>Files you upload here will be grouped together into a "Resource"</li>
-            <li>File size is limited to 1 GB per file</li>
+            <li>Files you upload here will be grouped together into a "Resource" in HydroShare.</li>
+            <li>File size is limited to 1 GB per file for direct browser upload from your local disk.</li>
+            <li>For file size larger than 1GB, you can create an iRODS account in HydroShare user server

```

Comment:
I suggest the following wording: Files larger than 1 GB can be imported directly from any iRODS account.  You can create a HydroShare iRODS account from your user profile page (link to user profile page). See help (link to help page) for information on working with iRODS directly.

This then requires writing an iRODS help file that I would like @rayi113 to take on.